helsingborg-stad/F-AI
# feat/prompt-playground

Enables prompt-playground in chainlit

## How to test 
Just run the chainlit app as usual, try write 'asdf'  should output same scoring values as before. Press the white bug (below message) in order to show prompt plauground.

## Changes
- add: model settings
- add: chat prompt class 
- add: factory methods for creating streams
- add: ChatStream template
- add: ScoreStream template
- factor: main loop & qa sheet script
- removed: collection.py (new logic & configuration can be found in chat dir)
